src/views/Invite/Url.js

```diff
@@ -0,0 +1,106 @@
+import React, { useCallback, useState } from 'react';
+import { Grid, ErrorText, Flex } from 'indigo-react';
+import { get } from 'lodash';
+import { FORM_ERROR } from 'final-form';
+
+import useCopiable from 'lib/useCopiable';
+import useInviter from 'lib/useInviter';
+
+import ProgressButton from 'components/ProgressButton';
+import { ForwardButton } from 'components/Buttons';
+import NeedFundsNotice from 'components/NeedFundsNotice';
+
+import { hasErrors } from 'form/validators';
+import { WARNING } from 'form/helpers';
+const STATUS = {
+  INPUT: 'INPUT',
+  SENDING: 'SENDING',
+  SUCCESS: 'SUCCESS',
+  FAILURE: 'FAILURE',
+};
+
+const InviteUrl = ({ setSubmitting }) => {
+  const { txStatus, needFunds, invites, generateInvites } = useInviter();
+
+  const [error, setError] = useState();
+
+  const generateInvite = useCallback(async () => {
+    setSubmitting(true);
+    const { errors } = await generateInvites(1);
+    if (hasErrors(errors)) {
+      setError(errors);
+    }
+    setSubmitting(false);
+  }, [generateInvites, setSubmitting]);
+
+  const renderGenerateButton = useCallback(() => {
+    if (error) {
+      return (
+        <Grid.Item full className="mt4 p3 bg-red3 white f5">
+          Error generating invite
+        </Grid.Item>
+      );
+    }
+    switch (txStatus) {
+      case STATUS.INPUT:
+        return (
+          <Grid.Item
+            solid
+            full
+            className="mt4"
+            as={ForwardButton}
+            onClick={() => generateInvite()}>
+            Generate Invite URL
+          </Grid.Item>
+        );
+      case STATUS.SENDING:
+        return (
+          <Grid.Item
+            full
+            as={ProgressButton}
+            className="mt4"
+            solid
+            disabled
+            success
+            progress={0.5}>
+            Generating Invite URL
+          </Grid.Item>
+        );
+      default:
+        return null;
+    }
+  }, [txStatus, error, generateInvite]);
+
+  const url = `https://bridge.urbit.org/#${get(invites, '[0].ticket', '')}`;
```

Comment:
This includes the leading `~`. We've traditionally (with the emails) been sending out invites in the format of `bridge.urbit.org/#some-ticket`, without leading `~`. I think it looks cleaner, and would remove it for the sake of consistency, but don't feel super strongly.

Reply:
Agree with @Fang- here, but that's my only quibble.

Reply:
fixed